### ISSUE TITLE
Multi-Day events + event limit + multiple event sources cause an exception

### ISSUE THREAD

**Issue by OClyde** [fullcalendar/fullcalendar#4585]
## Preconditions
1. The calendar must contain events spanning at least two days.
2. There must be more than one event source.
3. An event limit must be set that is cropping down events.

## Codepen to reproduce
https://codepen.io/OrangeClyde/pen/NJZNNE

## Steps to reproduce
1. Open the codepen.
2. Open the dev console.
3. Hit the "next" button.

Note the exception emitted in the console.

----
I'm a little bit in a rush right now, but I hope all the elementary information is in. Please do not hesitate to ask for additional information in case I missed something, will also have a second look later on!

**Comment by acerix** [comment]
Thanks for the detailed report, confirmed. I didn't find a workaround besides turning off the event limit.

**Comment by arshaw** [comment]
I'm trying to reproduce this, but I can't seem to make the exception show up in the console. is it still happening for you guys?

maybe it's dependent on the current date

**Comment by OClyde** [comment]
Yes it's still happening for me, basically every time I open the codepen, open the console and hit the "next" button - must be open before the first button hit because it's broken afterwards and won't happen again though, but as I can't change the timeframe at all then I suppose you would have recognized that as well.

So for you the codepen works just fine? Actually can't imagine how this could happen, that's weird!

**Comment by acerix** [comment]
I tested again and now don't get the exception on the first click, but if I keep clicking next, I get the exception at November.  

```
main.min.js:6 Uncaught (in promise) TypeError: Cannot read property 'parentNode' of undefined
    at c (main.min.js:6)
```

**Comment by OClyde** [comment]
Checked it again, the example I prepared wasn't ideal though I guess as I chose a 24 hour timeframe for the multi-day spanning when generating those dates, nevertheless this wasn't procuding the error at any point in time but only in specific ones (which could be a hint in itself, but still haven't found the time to dig deeper even though I'd love to :D).

Nevertheless, adjusted the example dates now to generate one date per day spanning across 60 hours, which should now be a case that reproduces the error on first hit.

Hope that helps - in case it doesn't please just let me know!

**Comment by arshaw** [comment]
thanks, i can now reproduce the error in the codepen

**Comment by arshaw** [comment]
this is fixed in v4.1.0

**Comment by OClyde** [comment]
Thanks a lot, that's awesome!
